rails/react-components/src/library/components/signup/basic_data_form.tsx

```diff
@@ -99,7 +99,7 @@ export default class BasicDataForm extends React.Component<any, any> {
               <dt>Password</dt>
               <dd><TextInput name="password" placeholder="" type="password" required validations={`minLength:${MIN_PASSWORD_LENGTH}`} validationError={PASS_TOO_SHORT} /></dd>
               <dt>Confirm Password</dt>
-              <dd><TextInput name="password_confirmation" placeholder="" type="password" required validations={{passwordMatchValidator: this.passwordMatchValidator}} validationError={PASS_NOT_MATCH} /></dd>
+              <dd><TextInput name="password_confirmation" placeholder="" type="password" required validations={{ passwordMatchValidator: this.passwordMatchValidator }} validationError={PASS_NOT_MATCH} /></dd>
             </dl>
```

Comment:
Changes in this file are the result of the `npm run lint:style:fix` that I ran before pushing my updates.